### PR TITLE
📚 add web support

### PR DESCRIPTION
**Pull request by A-Tokyo**
- 📚 add web support

NOTE:
This adds raw web support (ie: react not react native web). In a future PR I will try to add react-native web support as well.

### CODE REVIEW

**Review by mikehardy — CHANGES_REQUESTED** [review]
For a module called "react-native-something" I'm compelled to wait until there is react-native-web support :-), if this PR is just adding docs and relies on your library that seems fine but the doc could be added on your side

**Comment by A-Tokyo** [comment]
@mikehardy Thanks for the quick feedback ( : 

The purpose of this PR is to mainly provide a centralized place with all the docs related to implementing apple authentication (web/native/backend).

I opened it based on our experience implementing apple auth, the work felt stretched since there was barely any documentation back then. Once I noticed android came out and we easily implemented it, I wanted to ease the road for future developers ensuring that once they hit one doc, they can easily implement the full process in a day.

I have already added the documentation on my side ⚡️

I will try to do a rn-web project soon, but unrelated to this PR. So, this PR can be safely closed or merged. Please handle as you see fit

File: README.md [hunk 1]
```diff
@@ -221,6 +221,59 @@ async function onAppleButtonPress() {
 }
 ```
 
+
+### Web
```
Comment (mikehardy) [inline README.md:225]:
```suggestion
### Web (not react-native-web, but that may come as a follow-on, this is pure web at the moment)
```

**Review by mikehardy — APPROVED** [review]
Ok - I am pro-developer in that I understand especially in the react-native ecosystem it is really hard to pull together a multi-platform solution at the moment (where multi-platform is more than just android and ios) so to move that forward with personal action I'll merge this - I did make it explicit in the document that it was pure web not react-native-web though

Would love to have a react-native-web implementation if possible :-)

Cheers

**Comment by A-Tokyo** [comment]
Sounds great ( :

I added rn-web to my todos and will work on it soon 💪 

Cheers

**Pull request merged**